lib/hammer_cli_katello/sync_plan.rb

```diff
@@ -18,6 +18,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
 
       output ListCommand.output_definition do
         field :description, _("Description")
+        field :enabled, _("Enabled")
```

Comment:
I'd recommend adding the 'enabled' to the ListCommand vs Info.  This will allow it to be shown on both the list and info commands.  Also, having it on the List command makes it consistent with the UI which also shows that info.